render/render.cpp

```diff
@@ -104,6 +108,42 @@ void render_text(gfx::IPainter &painter, layout::LayoutBox const &layout, dom::T
     painter.draw_text(layout.dimensions.content.position(), text->text, font, font_size, color);
 }
 
+void render_borders(gfx::IPainter &painter, layout::LayoutBox const &layout) {
+    // TODO(mkiael): Handle a lot more border styles
+    auto color = style::get_property(*layout.node, "color"sv).value_or(kDefaultColor);
+    auto border_box = layout.dimensions.border_box();
+    auto const &border_size = layout.dimensions.border;
+    if (border_size.left > 0) {
+        geom::Rect border{border_box.x,
+                border_box.y + border_size.top,
```

Comment:
(0,0) is the top left corner, so this will start the border further down than the box it's a border for. I guess you could use `Rect::expanded(BorderSize)` to get the top/bottom/left/right points?

Reply:
Yea, doing like that makes vertical borders render correct when there is no horizontal borders.